pkg/kritis/violation/strategy.go

```diff
@@ -68,13 +68,13 @@ func (a *AnnotationStrategy) HandleViolation(image string, pod *v1.Pod, violatio
 		return nil
 	}
 	// Now, construct labels and annotations
-	var labelValue string
-	var annotationValue string
+	labelValue := constants.InvalidImageSecPolicyLabelValue
+	annotationValue := fmt.Sprintf("found %d CVEs.", len(violations))
 	for _, v := range violations {
 		if v.Violation == securitypolicy.UnqualifiedImageViolation {
-			labelValue = constants.InvalidImageSecPolicyLabelValue
+			annotationValue += string(v.Reason)
```

Comment:
func FixesNotAvailableViolationReason(image string, vulnz metadata.Vulnerability) Violation {
	return Violation(fmt.Sprintf("found CVE %s in %s which has fixes available", vulnz.CVE, image))
}

So, with the new change it will 
"found 50 CVEs, <image> is not a fully qualified image"